mappings/net/minecraft/client/render/chunk/ChunkRendererRegion.mapping

```diff
@@ -1,10 +1,10 @@
 CLASS net/minecraft/class_853 net/minecraft/client/render/chunk/ChunkRendererRegion
 	FIELD field_4481 offset Lnet/minecraft/class_2338;
-	FIELD field_4482 zSize I
+	FIELD field_4482 sizeZ I
 	FIELD field_4483 chunks [[Lnet/minecraft/class_2818;
-	FIELD field_4484 ySize I
+	FIELD field_4484 sizeY I
 	FIELD field_4485 fluidStates [Lnet/minecraft/class_3610;
-	FIELD field_4486 xSize I
+	FIELD field_4486 sizeX I
```

Comment:
Hmm, for these ones I am not quite sure. We don't usually view these 3 sizes as a whole (we don't use them together in a 3 vector or pass them together into a method)

Reply:
After thinking, this rename is fine. We can just view them as a 3-vector representing a size, and that name is very good.